internal/dnsforward/config.go

```diff
@@ -165,6 +165,9 @@ type TLSConfig struct {
 	cert tls.Certificate
 	// DNS names from certificate (SAN) or CN value from Subject
 	dnsNames []string
+
+	// ciphers specified by user
+	TLSCiphers []string `yaml:"tls_ciphers" json:"-"`
```

Comment:
1.  Please follow the [Go conventions on documenting](https://go.dev/doc/comment).  Unfortunately, a lot of older code doesn't follow them, but we strive to fix that and write all new code with the conventions in mind.

2.  The documentation here should mention that if the slice is empty or nil, the default ciphers are used.

3.  I feel like a better name would communicate that this field should generally not be used.  Something like `override_tls_ciphers`.

Reply:
added a function `getTLSCiphers()`